src/platform/switch/switch_ui.cpp

```diff
@@ -241,6 +292,16 @@ NxUi::NxUi(int width, int height, const Game_ConfigVideo& cfg) : BaseUi(cfg)
 #endif
 	SetIsFullscreen(true);
 
+	u32 hosVersion;
+	hosVersion = hosversionGet();
```

Comment:
Imo "auto hosVersion =" is safer here. DKP removed that u32 etc. From Wii and they love breaking stuff because they can. 
So bet this will be killed for switch in a few months :)